MapboxVisionAR/Render/ARScene/ConcreteNodes/ARCameraNode/ARCameraNode.swift

```diff
@@ -0,0 +1,70 @@
+import simd
+
+class ARCameraNode: ARNode {
+    // MARK: - Properties
+
+    /// Type of node. Always returns `rootNode`.
+    private(set) var nodeType: ARNodeType
+    /// Underlying AR entity.
+    var entity: AREntity?
+    /// Describes position of the node in the node hierarchy.
+    var relations: NodeRelations
+    /// Describes transformation between coordinate systems.
+    var geometry: NodeGeometry
+
+    var needsUpdateProjection = true
+    var nearClipPlane: Float = 0.01 {
```

Comment:
If you chose to comment everything why is this without a comment?

Reply:
I'm sorry for that. This particular code is a subject to change (see description of PR). This is why the implementation/comments here are a bit incomplete - just wanted to avoid duplication of work. Please see a corresponding issue TBD (comments will be added is scope of it).